File: liam code/fib.py
def fibonacci(n):
    if n == 1 or n == 2:
        return 1
    
    fib = [0] * (n+1)
    fib[1] = 1
    fib[2] = 1
    
    for i in range(3, n+1):
        fib[i] = fib[i-1] + fib[i-2]
        
    return fib[n]
print(fibonacci(int(input().strip())))

File: liam code/fib.js
var fibonacci = function(n) {
    if (n === 1 || n === 2) {
        return 1
    }

    let fib = new Array(n + 1).fill(0)
    fib[1] = 1
    fib[2] = 1

    for(let i = 3; i < n+1; i++) {
        fib[i] = fib[i-1] + fib[i-2]
    }

    return fib[n]
}

console.log(fibonacci(4))